src/components/NewTodoForm.js

```diff
@@ -0,0 +1,32 @@
+import PropTypes from 'prop-types'
+import { useState } from 'react'
+import Todos from './../todos'
+
+const NewTodoForm = ({ notifyChange }) => {
+    const [newTodoTitle, setNewTodoTitle] = useState('')
+
+    const createTodo = () => {
+        Todos.create({ title: newTodoTitle }).then(() => {
+            setNewTodoTitle('')
+            notifyChange()
+        })
+    }
+
+    return (
+        <>
+            <input
+                type='text'
+                placeholder='What needs to be done'
+                value={newTodoTitle}
+                onChange={(event) => { setNewTodoTitle(event.target.value) }}
+            />
+            <button onClick={createTodo}>Add</button>
+        </>
+    )
+}
```

Comment:
This should have been a form, since you already have to wrap the two tags in something. Also, it's a best practice to use labels for the inputs.